BASE/.vsts/linux-build.yml

```diff
@@ -14,17 +14,17 @@ steps:
 - task: UseDotNet@2
   displayName: install dotnet core 2.1
   inputs:
-    version: 2.1.807
+    version: 2.1.811
```

Comment:
I thought I read that you could use a wildcard here to always use the latest version available.
something like `2.1.*`
That might save us from needing to update this file regularly. 
But then we would lose testing with specific versions.

What are your thoughts?

Reply:
we were using. i will try to check if we still need the rsync.

so:
1. checking if changing will break (i hope it won't)
2. remove the specific version and remove rsync as well (that means 2.1.* and 3.1.*)